internal/controller/ocirepository_controller_test.go

```diff
@@ -1867,7 +1878,7 @@ func TestOCIRepository_getArtifactURL(t *testing.T) {
 		{
 			name: "valid url with no reference",
 			url:  "oci://ghcr.io/stefanprodan/charts",
-			want: "ghcr.io/stefanprodan/charts",
+			want: "ghcr.io/stefanprodan/charts:latest",
```

Comment:
@stefanprodan @hiddeco the tests are passing locally, this is the only real change of a test case that I had to make that might be significant.